README.md

```diff
@@ -117,34 +115,18 @@ When placeholder search is set to `false`, no results appears when searching on
 { placeholderSearch : true } // default true
 ```
 
-### Pagination total hits
-
-The total (and finite) number of hits (default: `200`) you can browse during pagination when using the [pagination widget](https://www.algolia.com/doc/api-reference/widgets/pagination/js/) or the [`infiniteHits` widget](#-infinitehits). If none of these widgets are used, `paginationTotalHits` is ignored.<br>
-
-For example, using the `infiniteHits` widget, and a `paginationTotalHits` of 9. On the first search request 6 hits are shown, by clicking a second time on `load more` only 3 more hits are added. This is because `paginationTotalHits` is `9`.
-
-Usage:
-
-```js
-{ paginationTotalHits: 50 } // default: 200
-```
-
-`hitsPerPage` has a value of `20` by default and can [be customized](#-hitsperpage).
-
 ### Finite Pagination
 
 Finite pagination is used when you want to add a numbered pagination at the bottom of your hits (for example: `<< < 1, 2, 3 > >>`).
-To be able to know the amount of page numbers you have, a search is done requesting `paginationTotalHits` documents (default: `200`).
-With the amount of documents returned, instantsearch is able to render the correct amount of numbers in the pagination widget.
+
+It requires the usage of the [`Pagination` widget](#-pagination).
 
 Example:
 
 ```js
 { finitePagination: true } // default: false
 ```
 
-⚠️ Meilisearch is not designed for pagination and this can lead to performances issues, so the usage `finitePagination` but also of the pagination widgets are not recommended.<br>
```

Comment:
I think this disclaimer is still accurate... Or at least part of it.

Reply:
@gmourier do you think we should keep this?

Reply:
I would remove it to see how the feature is received, we still do something completely different.

- By default, it is limited to the first 1000 results, which should guarantee good performance.
- We removed the hack previously in place, which posed performance problems. According to the testers, the new solution solves this previous pain point.